src/views/List.css

```diff
@@ -98,19 +98,48 @@ section {
 	display: flex;
 	flex-direction: column;
 	align-items: center;
+	padding-top: 2em;
+	padding-bottom: 2em;
```

Comment:
The padding added here causes the implementation to move away from the design. We actually don't want any margin or padding within the search section—the boxes are meant to occupy the whole space. The padding should be within each box.